docs/installing/INSTALLING-LOCALLY.md

```diff
@@ -0,0 +1,32 @@
+# Installing locally
+* [Installing](#installing)
```

Comment:
Looking over this file, it doesn't look like the TOC matches the content of the file.